sdks/go/test/integration/wordcount/wordcount_test.go

```diff
@@ -30,16 +31,26 @@ import (
 
 func TestWordCount(t *testing.T) {
 	tests := []struct {
-		lines []string
-		words int
-		hash  string
+		lines                                                             []string
+		words                                                             int
+		hash                                                              string
+		smallWords                                                        string
+		lineLen                                                           string
+		smallWordsCount, lineLenCount, lineLenSum, lineLenMin, lineLenMax int64
```

Comment:
I was mistaken earlier. We can do something simpler for the Distribution metric, by using a metrics.DistributionValue.

So `lineLen metricsDistributionValue`

then we can configure things with `metrics.DistributionValue{1,1,3,3},`  etc

And just do a straight equality check on the value

`distributonValue != test.lineLen`